roles/provision-sync-app-apb/templates/sync-dashboard.json.j2

```diff
@@ -548,7 +548,7 @@
           "expr": "process_resident_memory_bytes{app=\"sync-app-{{ SYNC_APP_NAME }}\"}",
           "format": "time_series",
           "intervalFactor": 1,
-          "legendFormat": "Pod {{ sync_app_pod_name }}",
+          "legendFormat": "Pod {% raw %}{{kubernetes_pod_name}}{% endraw %}",
```

Comment:
oh, so I didn't fully understand the reason of this format in [my previous PR](https://github.com/aerogearcatalog/sync-app-apb/pull/13/files), sorry about that.
So in that case we could probably remove the [`sync_app_pod_name` variable](https://github.com/aerogearcatalog/sync-app-apb/pull/13/files#diff-61ac620d7d20ca9bc92920918bd0982cR169) as well when it's not needed anymore?

Reply:
+1 removed it now